knative-operator/hack/update-deps.sh

```diff
@@ -20,6 +20,7 @@ OCP_VERSION="release-4.6"
 FLOATING_DEPS=(
   "github.com/openshift/api@${OCP_VERSION}"
 
+  "knative.dev/eventing@${KN_VERSION}"
```

Comment:
please add eventing-contrib too as it's now part of go.mod as well.

Reply:
yeah sure.